src/interface_r/scripts/gen_wrappers.R

```diff
@@ -49,51 +49,51 @@ write_wrapper("h2o4gpu$GradientBoostingRegressor",
             class_tags = 'c("regressor")',
             file_name = file_name)
 
-write_wrapper("h2o4gpu$LinearRegression",
-            r_function = "h2o4gpu.linear_regressor",
-            nullable_int_params = common_nullabl_int_params,
-            class_tags = 'c("regressor")',
-            file_name = file_name)
-
-write_wrapper("h2o4gpu$LogisticRegression",
-            r_function = "h2o4gpu.logistic_regressor",
-            nullable_int_params = common_nullabl_int_params,
-            class_tags = 'c("classifier")',
-            file_name = file_name)
-
-write_wrapper("h2o4gpu$Lasso",
-            r_function = "h2o4gpu.lasso_regressor",
-            nullable_int_params = common_nullabl_int_params,
-            class_tags = 'c("regressor")',
-            file_name = file_name)
-
-write_wrapper("h2o4gpu$Ridge",
-            r_function = "h2o4gpu.ridge_regressor",
-            nullable_int_params = common_nullabl_int_params,
-            class_tags = 'c("regressor")',
-            file_name = file_name)
-
-write_wrapper("h2o4gpu$ElasticNet",
-            r_function = "h2o4gpu.elastic_net_regressor",
+# write_wrapper("h2o4gpu$LinearRegression",
+#             r_function = "h2o4gpu.linear_regressor",
+#             nullable_int_params = common_nullabl_int_params,
+#             class_tags = 'c("regressor")',
+#             file_name = file_name)
+
+# write_wrapper("h2o4gpu$LogisticRegression",
+#             r_function = "h2o4gpu.logistic_regressor",
+#             nullable_int_params = common_nullabl_int_params,
+#             class_tags = 'c("classifier")',
+#             file_name = file_name)
+
+# write_wrapper("h2o4gpu$Lasso",
+#             r_function = "h2o4gpu.lasso_regressor",
+#             nullable_int_params = common_nullabl_int_params,
+#             class_tags = 'c("regressor")',
+#             file_name = file_name)
+# 
+# write_wrapper("h2o4gpu$Ridge",
+#             r_function = "h2o4gpu.ridge_regressor",
+#             nullable_int_params = common_nullabl_int_params,
+#             class_tags = 'c("regressor")',
+#             file_name = file_name)
```

Comment:
Are you planning to support these going forward in H2O wrappers? If yes, could you put a comment here? If no, please simply remove them for now.

Reply:
I am not sure yet so I will add a note.

Reply:
Another question is: if sklearn is a requirement to build the package, should we simply keep it as it is since it's already installed and we can still keep the wide coverage of types of models.

Reply:
I wouldn't think that's not ideal since the sklearn requirement is needed for a specific reason: to be compatible with current scikit like pipelines in Python. Doesn't really apply in this scenario.

Reply:
A good rule of thumb is if the class has a `backend` argument, then it is not needed for the R api as we wouldn't expect an R user to want to use scikit.

Reply:
okay I see. Thanks for the clarification. Ping us when this is ready for review.